db/migrate/20210422171642_add_indexes_to_comments.rb

```diff
@@ -0,0 +1,15 @@
+class AddIndexesToComments < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def up
+    add_index :comments, :commentable_type, algorithm: :concurrently unless index_exists?(:comments, :commentable_type)
+    add_index :comments, :deleted, algorithm: :concurrently unless index_exists?(:comments, :deleted)
+    add_index :comments, :hidden_by_commentable_user, algorithm: :concurrently unless index_exists?(:comments, :hidden_by_commentable_user)
```

Comment:
non blocking and for future reference: Rails 6.1 has added `if_not_exists:true` for `add_index` and `if_exists: true` for `remove_index`

Reply:
Haha you beat me to it 😅  https://github.com/forem/forem/pull/13476#discussion_r618605158